hw3/indexhw3.html

```diff
@@ -0,0 +1,135 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<title>hw3</title>
+	<link rel="stylesheet" href="css/reset.css">
+	<link rel="stylesheet" href="css/fonts.css">
+	<link rel="stylesheet" href="css/stylehw3.css">
+	<link rel="stylesheet" href="css/response.css">
+</head>
+<body>
+	<header class="header">
+		<div class="header__wrapper">
+			<span class="header__logo">Global</span>
+			<ul class="header__menu">
+				<li class="header__menu__item"><a href="#" class="header__menu__link">Home</a></li>
```

Comment:
incorrect! There is no elements in elements. No way to make double __ in className. Please split you blocks into blocks. 
header
header__wrapper
header__logo
header-menu
header-menu__item
header-menu__link